client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -140,7 +140,7 @@ const Sidebar = ( { sidebarDomain, siteSlug, submit, goNext, goToStep, flow }: S
 						) }
 					</div>
 					{ sidebarDomain?.isWPCOMDomain && (
-						<a href={ `/domains/add/${ siteSlug }` }>
+						<a href={ `/domains/add/${ siteSlug }?is-launchpad-step=true` }>
```

Comment:
We'll likely need to fix the failing test to be less strict and ignore the query args:

https://github.com/Automattic/wp-calypso/blob/4e8a58dac366528d488281140711b29c8bcd2970/client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/sidebar.tsx#L92